File: sub-model/src/main/java/org/github/base/entity/QrtzFiredTriggersEntity.java
package org.github.base.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import com.baomidou.mybatisplus.annotation.IdType;
import org.github.base.AbstractEntity;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableField;
import lombok.*;

/**
 * <p>
 * 
 * </p>
 *
 * @author JYD_XL
 * @since 2019-12-12
 */
@Setter
@Getter
@EqualsAndHashCode(callSuper = true)
@TableName("qrtz_fired_triggers")
public class QrtzFiredTriggersEntity extends AbstractEntity {
    /** UID */
    private static final long serialVersionUID = 1L;

    @TableId(value = "sched_name", type = IdType.ASSIGN_UUID)
    private String schedName;

    @TableField("entry_id")
    private String entryId;

    @TableField("trigger_name")
    private String triggerName;

    @TableField("trigger_group")
    private String triggerGroup;

    @TableField("instance_name")
    private String instanceName;

    @TableField("fired_time")
    private Long firedTime;

    @TableField("sched_time")
    private Long schedTime;

    @TableField("priority")
    private Integer priority;

    @TableField("state")
    private String state;

    @TableField("job_name")
    private String jobName;

    @TableField("job_group")
    private String jobGroup;

    @TableField("is_nonconcurrent")
    private String isNonconcurrent;

    @TableField("requests_recovery")
    private String requestsRecovery;
}

File: sub-model/src/main/java/org/github/base/entity/QrtzFiredTriggersEntity.kt
package org.github.base.entity

import com.baomidou.mybatisplus.annotation.TableName
import org.github.base.AbstractEntity

/**
 * <p>
 * 
 * </p>
 *
 * @author JYD_XL
 * @since 2019-11-19
 */
@TableName("qrtz_fired_triggers")
class QrtzFiredTriggersEntity: AbstractEntity() {

    var schedName: String? = null

    var entryId: String? = null

    var triggerName: String? = null

    var triggerGroup: String? = null

    var instanceName: String? = null

    var firedTime: Long? = null

    var schedTime: Long? = null

    var priority: Int? = null

    var state: String? = null

    var jobName: String? = null

    var jobGroup: String? = null

    var isNonconcurrent: String? = null

    var requestsRecovery: String? = null

}
